content/agent/proxy.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Proxy Configuration
+kind: documentation
+customnav: agentnav
+---
+## Why use a Proxy
+
+If your network configuration restricted outbound traffic, proxy all agent traffic through one or several hosts that have more permissive outbound policies.
+
+A few options are available to send traffic to Datadog over SSL/TLS for
+hosts that are not directly connected to the internet.
+
+1. Using the agent as a proxy (for **up to 16 agents** per proxy)
+2. Using a web proxy (e.g. Squid, Microsoft Web Proxy) that is already deployed in your network
+3. Using HAProxy (if you want to proxy **more than 16-20 agents** through the
+same proxy)
+
+## Step-by-step Guides
+
+### Using the Agent as a Proxy
+
+1. Designate one node **running datadog-agent** as the proxy.  
+    In this example assume that the proxy name is `proxy-node`. This node **must** be able to reach `https://app.datadoghq.com`.
+
+2. Verify SSL connectivity on `proxy-node`  
+    ```
+    curl -v https://app.datadoghq.com/account/login 2>&1 | grep "200 OK"
+    ```
+
+3. Allow non-local traffic on `proxy-node` by changing the following line in `/etc/dd-agent/datadog.conf`.  
+     `# non_local_traffic: no` should read `non_local_traffic: yes`.
+
+4. Make sure `proxy-node` can be reached from the other nodes over port 17123. Start the agent on the `proxy-node` and run on the other nodes:
+
+    `curl -v http://proxy-node:17123/status 2>&1 | grep "200 OK"`
+
+5. Update non-proxy nodes to forward to `proxy-node`. Simply change the following line in `/etc/dd-agent/datadog.conf` from:
```

Comment:
we are listing a specific path a few times, but the path is different (according to our docs for at least 2 platforms (mac and windows)), should we rephrase to the datadog.conf file (look in the agent docs for the location on your platform)?

Reply:
:+1: